draft-ietf-ppm-dap.md

```diff
@@ -2125,8 +2126,8 @@ The Helper constructs an `AggregationJobResp` message (see
 {{aggregation-helper-init}}) with each prep step. The order of the prep steps
 MUST match the Leader's `AggregationJobContinueReq`.
 
-The Helper responds to the Leader with HTTP status 200 OK, a body consisting
-of the `AggregationJobResp`, and the media type
+The Helper responds to the Leader with HTTP status 202 Accepted, a body
+consisting of the `AggregationJobResp`, and the media type
```

Comment:
This looks like a bug fix. What do you want to do for interop? Should we cut a new draft so folks can implement this? Or are we OK with implementing 200 OK here?

Reply:
This is indeed a bugfix (or, well, it resolves an ambiguity in the spec).

I think we should eventually cut a bugfix draft w/ this and any other small fixes we find during implementation. For DAP-13 interop testing we can handshake on the appropriate status; this PR switches to 202 everywhere, and that's what I'm currently implementing in Janus, but I don't have a strong opinion either way on the correct status to use -- I'd be happy enough with just about anything as long as we're consistent.

Reply:
It would be nice to avoid cutting a draft if we can avoid it. If you think the intent is clear enough, then I think we should treat this as non-breaking. My suggestion would be to merge, update the change log, cut a draft, and inform the mailing list.

Reply:
We could also wait to see if more changes are needed after we've all wrapped up our implementations. I don't think we've caught any bugs in daphne yet.

Reply:
IMO, let's wait until we think our implementations are finished. Then we'll be in a better position to decide if we need to cut a new draft or just inform folks. In any case, I'll wait a few days to let anyone object & then merge this; heads-up that this specifies 202 Accepted as the correct response code for a Helper receiving an aggregation continuation request.

I don't think the intent is clear, however; the text for Helper aggregation continuation currently says that the Helper should respond with a 202 Accepted in one place, and a 200 OK in another. I think this is something that should be cleaned up before RFC, one way or another.